src/presentation/pages/Settings.vue

```diff
@@ -55,6 +62,15 @@ async function userLogout() {
   });
 }
 
+/**
+ * Deletes tool from the user
+ *
+ * @param toolId - id of the tool
+ */
+async function userDeleteTool(toolId: string) {
+  await deleteTool(toolId);
```

Comment:
what will happen next? Button text should be changed to "Uninstalled" and it should became inactive

Reply:
and after reload of the page it should be removed from user settings?

Reply:
yes